pkg/endpointmanager/conntrack.go

```diff
@@ -83,9 +83,13 @@ func RunGC(e *endpoint.Endpoint, isIPv6 bool, filter *ctmap.GCFilter) {
 }
 
 // EnableConntrackGC enables the connection tracking garbage collection.
-func EnableConntrackGC(ipv4, ipv6 bool) {
+func EnableConntrackGC(ipv4, ipv6 bool, gcinterval int) {
 	go func() {
-		sleepTime := time.Duration(GcInterval) * time.Second
+		if gcinterval < MinGcInterval {
+			gcinterval = MinGcInterval
+			log.Warnf("Setting conntrack garbage collector interval to its minimum value(%d seconds)", gcinterval)
+		}
+		sleepTime := time.Duration(gcinterval) * time.Second
```

Comment:
the `time.Duration` cast is unnecessary, I think. We do have it all over but it would be nice to not add more :/

Reply:
I get invalid operation: gcinterval * time.Second (mismatched types int and time.Duration).

Reply:
hmm. I guess I was wrong. The other instances I have in mind must have been literal numbers, and they were converted appropriately. Ignore me!